community/learn/hasura/graphql-tutorial-backend/tutorial-site/content/custom-business-logic/3-write-event-webhook.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Write event webhook"
+---
+
+Now let's move to the second use-case of Sending an email when a user registers on the app.
+
+When the user registers on the app using Auth0, we insert a new row into the `users` table to keep the user data in sync. Remember the rule we wrote to make a mutation?
+
+This is an `insert` operation on table `users`.
+
+Now we are going to capture this insert operation to trigger our event.
```

Comment:
Add a second line which says: the payload for each event is mentioned here: <doc-link>